### ISSUE TITLE
Toggle controls inaccessible to VoiceOver and keyboard users creating significant accessibility barrier

### ISSUE THREAD

**Issue by ssawczyn** [aaronpk/OwnYourSwarm#50]
As a VoiceOver user, I want to be able to change the various toggleable settings.  Unfortunately, as these do not appear to be standard HTML controls, VoiceOver is not able to identify their current state.  Because these are not standard HTMl controls, users who rely on the keyboard rather than a mouse are also unable to toggle these controls as there's no way for them to get focus.

While using native HTML controls such as checkboxes is generally preferable, it's possible to use ARIA to convey a custom control's name/role/value to assistive technologies.  This article may provide additional context:  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role 

I'm really excited about this project and hope there's a way we can make the settings more accessible to VoiceOver and keyboard users.
